zcbor/zcbor.py

```diff
@@ -2768,8 +2756,18 @@ def parse_args():
 referencing types, since the C type cannot be self referencing.
 
 This script requires 'regex' for lookaround functionality not present in 're'.''',
-        formatter_class=RawDescriptionHelpFormatter)
+        formatter_class=RawDescriptionHelpFormatter,
+        parents=[parent_parser])
 
+    code_parser.add_argument(
+        "--default-max-qty", "--dq", required=False, type=int_or_str, default=3,
+        help="""Default maximum number of repetitions when no maximum
```

Comment:
nit: Most of these use `'''` in place of quotes, and couple of these like here aren't consistent. Feel free to ignore though.